evalai/utils/submissions.py

```diff
@@ -60,18 +63,29 @@ def pretty_print_my_submissions_data(submissions):
         echo(style("\nSorry, you have not made any submissions to this challenge phase.\n", bold=True))
         sys.exit(1)
 
+    if not start_date:
+        start_date = datetime.min
+
+    if not end_date:
+        end_date = datetime.max
+
     for submission in submissions:
-        date = convert_UTC_date_to_local(submission['submitted_at'])
-        # Check for empty method name
-        method_name = submission["method_name"] if submission["method_name"] else "None"
-        values = list(map(lambda item: submission[item], attributes))
-        values.append(date)
-        values.append(method_name)
-        table.append_row(values)
+        date = validate_date_format(submission['submitted_at'])
+        if (date >= start_date and date <= end_date):
+            # Check for empty method name
+            date = convert_UTC_date_to_local(submission['submitted_at'])
+            method_name = submission["method_name"] if submission["method_name"] else "None"
+            values = list(map(lambda item: submission[item], attributes))
+            values.append(date)
+            values.append(method_name)
+            table.append_row(values)
+    if len(table) == 0:
+        echo(style("\nSorry, no submissions made during this time period.\n", bold=True))
```

Comment:
`Sorry, no submissions were made during this time period.`

Reply:
@RishabhJain2018 Updated.